test/libsolidity/syntaxTests/array/calldata_resize.sol

```diff
@@ -4,4 +4,4 @@ contract C {
     }
 }
 // ----
-// TypeError: (75-83): Calldata arrays cannot be resized.
+// TypeError: (75-83): Member "length" is read-only and cannot be used to resize arrays.
```

Comment:
Should there be a test for ``x.push()`` to fail for calldata arrays?

Reply:
Now there is :-)